src/Share/Survey/SurveyHelper.cs

```diff
@@ -78,213 +97,62 @@ public static SurveyHelper GetInstance()
-
-        private void ModulePrompt(SurveyHelper helper, string moduleName, int majorVersion)
-        {
-            helper.LastPromptDate = Today;
-            helper.Modules[moduleName].LastActiveDate = CurrentDate;
-            helper.Modules[moduleName].ActiveDays += 1;
-        }
-
-        private bool ShouldAzPredictorPrompt(SurveyHelper helper, string moduleName, int majorVersion)
-            => _predictor.Equals(moduleName, StringComparison.OrdinalIgnoreCase)
-            && helper.Modules[moduleName].MajorVersion == majorVersion
-            && ((helper.Modules[moduleName].ActiveDays == _surveyTriggerCount && helper.Today <= Convert.ToDateTime(helper.Modules[moduleName].FirstActiveDate).AddDays(_countExpiredDays))
-                 || helper.Modules[moduleName].ActiveDays == _surveyTriggerCount + 1 && helper.Today == Convert.ToDateTime(helper.Modules[moduleName].LastActiveDate).AddDays(_delayForSecondPrompt)
-                 || helper.Modules[moduleName].ActiveDays == _surveyTriggerCount + 2 && helper.Today == Convert.ToDateTime(helper.Modules[moduleName].LastActiveDate).AddDays(_delayForThirdPrompt));
-
-        private void AzPredictorPrompt(SurveyHelper helper, string moduleName, int majorVersion)
-        {
-            helper.Modules[moduleName].LastActiveDate = CurrentDate;
-            helper.Modules[moduleName].ActiveDays += 1;
-        }
-
-        private bool ShouldModulePromptExpire(SurveyHelper helper, string moduleName, int majorVersion)
-            => helper.Modules[moduleName].MajorVersion == majorVersion
-            && (Modules[moduleName].ActiveDays == _surveyTriggerCount + 1 && helper.LastPromptDate == Convert.ToDateTime(Modules[moduleName].LastActiveDate) && helper.Today > helper.LastPromptDate.AddDays(_delayForSecondPrompt)
-                || Modules[moduleName].ActiveDays == _surveyTriggerCount + 2 && helper.LastPromptDate == Convert.ToDateTime(Modules[moduleName].LastActiveDate) && helper.Today > LastPromptDate.AddDays(_delayForThirdPrompt));
-
-        private void ModulePromptExpire(SurveyHelper helper, string moduleName, int majorVersion)
-        {
-            helper.Modules[moduleName].ActiveDays = 0;
-        }
-
-        private void MergeScheduleInfo(ScheduleInfo externalScheduleInfo)
-        {
-            DateTime externalLastPromptDate = Convert.ToDateTime(externalScheduleInfo?.LastPromptDate);
-            IDictionary<string, ModuleInfo> externalModules = new Dictionary<string, ModuleInfo>();
-            foreach(ModuleInfo info in externalScheduleInfo?.Modules)
-            {
-                externalModules[info.Name] = info;
-            }
-
-            HashSet<string> moduleNames = new HashSet<string>(Modules.Keys);
-            moduleNames.UnionWith(new HashSet<string>(externalModules.Keys));
-
-            foreach (string name in moduleNames)
+        private void InitialSurveyHelper(){
```

Comment:
Our coding style is to put { in a standalone line. Please remember to format the document after making any changes
```suggestion
        private void InitialSurveyHelper()
        {
```